### PR TITLE
Chore/setup rubocop for travis

### PR DESCRIPTION
**Pull request by ochagata**
Fixed most style violations without making any tests start failing.
Also, configured rakefile and travis.yml so that it will run rubocop and rspec on every commit.

### CODE REVIEW

**Comment by ochagata** [comment]
@bellycard/platform

@darbyfrey It's green

**Comment by darbyfrey** [comment]
This looks great! Thanks for the contributions @ochagata!

@bellycard/platform please review these changes and let me know if you see any issues here. 

+1 from me.

File: lib/napa/active_record_extensions/notifications_subscriber.rb [hunk 1]
```diff
@@ -1,17 +1,17 @@
 if defined?(ActiveRecord)
-  ActiveSupport::Notifications.subscribe 'sql.active_record' do |name, start, finish, id, payload|
+  ActiveSupport::Notifications.subscribe 'sql.active_record' do |_name, start, finish, _id, payload|
     if payload[:sql].downcase.match(/(select|update|insert|delete)(.+)/i)
       table, action = Napa::ActiveRecordStats.extract_sql_content(payload[:sql])
     end
 
     if table
       Napa::Stats.emitter.timing(
-        "sql.query_time",
```
Comment (ashtonthomas) [inline lib/napa/active_record_extensions/notifications_subscriber.rb]: you no like double quotes? Any reason in particular for this change?

[ref](https://viget.com/extend/just-use-double-quoted-ruby-strings#tldr)

File: lib/napa/cli/base/console.rb [hunk 2]
```diff
@@ -23,10 +22,9 @@ def console(environment = nil)
 
         Rack::Shell.init
 
-        $0 = "#{$0} console"
+        $0 = "#{$PROGRAM_NAME} console"
```
Comment (ashtonthomas) [inline lib/napa/cli/base/console.rb]: nice

File: lib/napa/cli/base/new.rb [hunk 3]
```diff
@@ -23,27 +22,22 @@ def new(name, path = nil)
...
-        def app_path
-          @path
-        end
+        attr_reader :name, :path
+        alias_method :app_name, :name
+        alias_method :app_path, :path
 
         def postgres?
-          %w[pg postgres].include?(options[:database])
+          %w(pg postgres).include?(options[:database])
```
Comment (ashtonthomas) [inline lib/napa/cli/base/new.rb]: :+1:

File: lib/napa/deprecations/application_api.rb [hunk 4]
```diff
@@ -1,8 +1,10 @@
 module Napa
   class Deprecations
+    LINK = 'https://github.com/bellycard/napa/blob/master/lib/napa/generators/templates/scaffold/app/apis/application_api.rb'
```
Comment (ashtonthomas) [inline lib/napa/deprecations/application_api.rb]: @darbyfrey this file doesn't seem to be alive anymore. Should remove this or has that file been moved somewhere else?

**Comment by ashtonthomas** [comment]
LGTM

File: lib/napa/deprecations/application_api.rb (hunk 4 above)
Comment (ochagata) [inline lib/napa/deprecations/application_api.rb]: Yeah, I noticed that as I made this change, it used to work when the link was committed but not anymore. Not sure what happened to it.

Comment (darbyfrey) [inline lib/napa/deprecations/application_api.rb]: Looks like the file got move to here: https://github.com/bellycard/napa/blob/master/lib/napa/cli/templates/new/app/apis/application_api.rb

**Comment by darbyfrey** [comment]
@ochagata, I'm going to merge this in a do a quick PR for that URL change.

**Pull request merged**